src/analyses/malloc_null.ml

```diff
@@ -164,7 +164,7 @@ struct
   let return_addr () = !return_addr_
 
   let return man (exp:exp option) (f:fundec) : D.t =
-    let remove_var x v = List.fold_right D.remove (to_addrs v) x in
+    let remove_var x v = List.fold_left (Fun.flip @@ D.remove) x (to_addrs v) in
```

Comment:
I think this should also just work:
```suggestion
    let remove_var x v = List.fold_left (Fun.flip D.remove) x (to_addrs v) in
```
If so, then the same goes for some following places as well.